pyeasee/easee.py

```diff
@@ -7,7 +7,7 @@
 import logging
 import ssl
 import time
-from typing import Any, AsyncIterator, Deque, Dict, List
+from typing import Any, AsyncIterator, Deque, Dict, List, Optional
```

Comment:
We will slowly improve type hints in this library and allow | as union operator. 
```suggestion
from typing import Any, AsyncIterator, Deque, Dict, List
```

Reply:
Adding `from __future__ import annotations` at top of imports should fix backward compatibility for a couple of python versions.